### DISCUSSION TITLE
can a user of a created map create a unique log in for themselves?

### DISCUSSION THREAD

**Discussion by rawnoxfam** [FacilMap/facilmap#289] [General]
hello! 

wondering as i'm trying to use facil map for a work project - it doesn't seem like there is any way to create a unique log in per user of a created map, is that correct? 

thank you!

**Comment by cdauth** [comment]
Right now there is no login functionality yet. What exactly would you need it for?

**Comment by rawnoxfam** [comment]
Thanks so much for replying. My company was trying to use it in a way such that multiple users could access the same map, and said users could suggest prospects for us to follow up on by creating markers and including some of the prospects' personal info in the description of the marker. Obviously we wouldn't want everyone accessing the map everyone's personal details so we wondered about a log-in system where they would only be able to read the descriptions of their own markers. I figured probably not possible! Does this seem out of question for future development? thanks so much for answering.

________________________________

**Reply by cdauth** [reply]
For now this is definitely not possible, but there are rough plans that should make this possible at some point. I just don't know when I will find the time to implement this, it could be several months or years.

Right now every map has exactly 3 links, one admin, one read-write and one read-only. The plan is to be able to configure a custom amount of links and configure their permissions in more detail. The configurable permissions would include being able to see or to modify only certain object types and/or only certain fields of those object types. In addition, specific permissions could be configured for the user who created an object. Right now, since there are no user accounts, this would involve storing some kind of anonymous cookie in the browser to identify the user.

Another plan is to introduce user accounts. They would serve as an optional feature to persist what is now stored in cookies (in particular collaborative map bookmarks) and be able to use it across devices. This would make the special “object creator” permissions much more useful.

I believe that once those two features are implemented, your use case would be possible.
